tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -3382,12 +3382,20 @@ SpirvInstruction *SpirvEmitter::doShortCircuitedConditionalOperator(
 
 SpirvInstruction *
 SpirvEmitter::doConditionalOperator(const ConditionalOperator *expr) {
```

Comment:
Maybe we can get rid of `doConditionalOperator` entirely now and just call `doConditional` with the right arguments, like you do for the intrinsics?

Reply:
Done